pkg/node/volumemgr.go

```diff
@@ -996,3 +1019,29 @@ func prepareDriveDescription(drive *drivecrd.Drive) string {
 		drive.Spec.SerialNumber, drive.Spec.NodeId, drive.Spec.Type,
 		drive.Spec.VID, drive.Spec.PID, drive.Spec.Size, drive.Spec.Firmware)
 }
+
+// isDriveSystem check whether drive is system
+// Parameters: path string - drive path
+// Returns true if drive is system, false in opposite; error
+func (m *VolumeManager) isDriveSystem(path string) (bool, error) {
+	devices, err := m.listBlk.GetBlockDevices(path)
+	if err != nil {
+		return false, err
+	}
+	return m.isRootMountpoint(devices), nil
+}
+
+// isRootMountpoint check whether devices has root mountpoint
+// Parameters: BlockDevice from lsblk output
+// Returns true if device has root mountpoint, false in opposite
+func (m *VolumeManager) isRootMountpoint(dev []lsblk.BlockDevice) bool {
```

Comment:
since dev it is a slice (not a single object) could you please rename dev to devs